.github/workflows/ci.yml

```diff
@@ -39,103 +32,179 @@ jobs:
             version: '7.0'
           - name: 'active_record'
             version: '7.1'
+          - name: 'active_record'
+            version: '7.2'
+          - name: 'active_record'
+            version: '8.0'
```

Comment:
Since AR8 is still failing, can we exclude it or put it under experimental for now so we can ship the main fix?  Then at least folks not using that feature can point to `master` and use Mobility on Rails 8 for most stuff.